public-ui/src/app/shared/components/rh-header/rh-header.tsx

```diff
@@ -0,0 +1,165 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+import * as React from 'react';
+
+import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
+import { menuItems } from 'app/shared/services/header-footer.service';
+import { environment } from 'environments/environment';
+
+interface State {
+  menuOpen: boolean;
+  searchTerms: string;
+  submenusOpen: Array<string>;
+}
+
+export class RhHeader extends React.Component<{}, State> {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      menuOpen: false,
+      searchTerms: '',
+      submenusOpen: []
+    };
+  }
+
+  toggleMobileSubmenu(title: string) {
+    const {submenusOpen} = this.state;
+    const updatedSubmenusOpen = submenusOpen.includes(title) ?
+      submenusOpen.filter(sub => sub !== title) :
+      [...submenusOpen, title];
+    this.setState({submenusOpen: updatedSubmenusOpen});
+  }
+
+  render() {
+    const {menuOpen, searchTerms, submenusOpen} = this.state;
+    return <React.Fragment>
+      <div className='cta-bar'>
+        <a className='login-btn'
+           href='https://workbench.researchallofus.org/login'
+           style={{width: 'auto'}}>
```

Comment:
Had to use an inline style here also since it was applying `width: 100%` from the footer CSS